tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Status.feature

```diff
@@ -117,3 +117,14 @@ Scenario: 10. Update an order status sets the complete date
     When the user sends the update order status request
     Then a response with status code 204 is returned
     And the order completed date is set
+
+@3928
+Scenario: 11. When an order is complete, an email is sent
+    Given the user creates a request to update the order status for the order with ID 'C000014-01'
+    And the user enters the 'order-status-complete' update order status request payload
+    When the user sends the update order status request
+    Then a response with status code 204 is returned
+    And only one email is sent
+    And the email sent contains the following information

```

Comment:
Unfortunately might need to split this assertion out. This email in the future will contain more than one attachment & I am not sure how to do this with the step as it is. I wonder if we can simplify the assertion of the attachment document to check each line. Any ideas? What do you think?

Could be raised as tech debt to get this story in.

Reply:
I think raise it as tech debt so we can merge the ordering in. I can raise it if you want. As tbh I did a similar version as when there are more headings ect, you will want to change how it is written